### ISSUE TITLE
Blank window when running electron app as forced process

### ISSUE THREAD

**Issue by hg421** [sandboxie-plus/Sandboxie#3198]
### Describe what you noticed and did

I have configured a certain Electron app (https://github.com/erkyrath/lectrote) to run as forced process in a standard isolation (yellow) box. However, starting it from an unsandboxed explorer window now results in the app showing a blank window (where UI elements, even if not visible, are still clickable):
<img width="453" alt="blankwindow" src="https://github.com/sandboxie-plus/Sandboxie/assets/78082344/b37c5d2c-1605-4c3b-ab66-568d8465bfc5">
This also happens with VS Code so presumably it's an Electron related issue.

The first affected Sandboxie version is 1.10.1; with version 1.10.0 there is no such problem.

Further notes:
- problem only occurs when running as forced process
- problem occurs when running from unsandboxed explorer window, but **not** when running from unsandboxed command prompt (even though it's forced into the box in both cases)
- problem seems to happen when the forced process is restarted from SbieSvc due to PCA job detection (that was modified in version 1.10.1)

### Affected program

Lectrote (https://github.com/erkyrath/lectrote) or VS Code

### Download link

https://github.com/erkyrath/lectrote

### Where is the program located?

The program is installed both inside and outside the sandbox.

### Expected behavior

App window displays properly

### What is your Windows edition and version?

Windows 11 22H2

### In which Windows account you have this problem?

A local account (Standard user).

### Please mention any installed security software

Windows Defender

### What version of Sandboxie are you running?

Sandboxie Plus 1.10.5

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### Is it a regression?

Since version 1.10.1

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

**Comment by offhub** [comment]
I cannot reproduce this with Sandboxie Plus 1.10.4/5. [Windows 10 (host)/Windows 11 (vm)]

Do you use any custom settings for Sandboxie?

**Comment by hg421** [comment]
> I cannot reproduce this with Sandboxie Plus 1.10.4/5. [Windows 10 (host)/Windows 11 (vm)]
> 
> Do you use any custom settings for Sandboxie?

I don't think so, here are the relevant sections of my Sandboxie.ini:
```
[GlobalSettings]
DefaultBox=DefaultBox
FileRootPath=\??\%SystemDrive%\Sandbox\%USER%\%SANDBOX%
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
Template=WindowsRasMan
Template=WindowsLive
Template=OfficeLicensing
Template=OfficeClickToRun
Template=Edge_Fix
Template=7zipShellEx

[TestBox]
Enabled=y
BlockNetworkFiles=y
BorderColor=#02f6f6,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=10
UseSecurityMode=n
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=y
ForceProcess=Lectrote.exe
```


Are you getting the SbieSvc restart when running it from explorer? It's indicated in the trace log like this
<img width="487" alt="restart-trace-log" src="https://github.com/sandboxie-plus/Sandboxie/assets/78082344/7cd606a3-894b-4b40-9986-722f723ea402">

**Comment by offhub** [comment]
> Are you getting the SbieSvc restart when running it from explorer? It's indicated in the trace log like this

Yes,

Does the program open correctly after adding the `NoRestartOnPAC=y` setting to the sandbox configuration?

https://github.com/sandboxie-plus/Sandboxie/assets/6871698/1d0436d2-ce04-49d0-accb-2d9a2d41ed3e

**Comment by hg421** [comment]
> Does the program open correctly after adding the `NoRestartOnPAC=y` setting to the sandbox configuration?

Yes, that's a viable workaround (it basically restores the 1.10.0 behaviour again, I guess).

But there seems to be indeed some incompatibility with Electron and SbieSvc restarting, at least on some machines (maybe only Windows 11 and with a real GPU, since it works in your VM?) I'm using a Windows tablet with Intel graphics, btw.
